Services/AuthenticationService/Validators/UserEmailPasswordValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using DTO;
+using FluentValidation;
+
+namespace AuthenticationService.Validators
+{
+    public class UserEmailPasswordValidator : AbstractValidator<UserEmailPassword>
+    {
+        public UserEmailPasswordValidator()
+        {
+            RuleFor(user => user.Email)
+                .NotEmpty()
+                .WithMessage("Email address must not be empty.")
+                .EmailAddress()
+                .WithMessage("Email address must be in valid format.");
+
+            RuleFor(user => user.Password)
+                .NotEmpty()
+                .WithMessage("Password must not be empty.")
+                .Length(5, 500)
+                .WithMessage("Password's length must be from 5 to 500 symbols.")
+                .Matches(@"^[a-zA-Z0-9]+$")
```

Comment:
And how about spaces, symbols? It is just a suggestion. 

Also, I think 500 is a little more that people needed

Reply:
Why would anyone need password like _"  /-==/%/*[]^][ [ ]\ [  \"_, it's not cool :(

Reply:
I wanted to set 200-300 characters long password once, and when the service's response was like "password can't be longer than 50 symbols", I was a bit disappointed actually, because it would be fun otherwise

Reply:
Wow, seriously? Okay :) 
Still don't understand why, but okay. 

Yes, that's password is bad, but how about "my password" or "kalinka-malinka" or sth like that

Reply:
And also, I think that most of people have small passwords. Like.. 98 % will have password less than 50 symbols. And you will always have memory for 450 symbols

Reply:
And do you have checks for length of email?

Reply:
By the way, some people use password generators with special symbols. In our _trading platform_ a strong password is essential

Reply:
> And do you have checks for length of email?

Whaaaat, where?

Reply:
> And also, I think that most of people have small passwords. Like.. 98 % will have password less than 50 symbols. And you will always have memory for 450 symbols

What? Validation != giving memory

Reply:
> By the way, some people use password generators with special symbols. In our _trading platform_ a strong password is essential

Well, agree, removed the limitation.